packages/angular/projects/ui-angular/src/lib/primitives/button/button.component.ts

```diff
@@ -17,6 +18,10 @@ export class ButtonComponent implements OnInit {
   @HostBinding('attr.data-variation') variationAttr: string;
   @HostBinding('style.font-weight') fontWeightAttr: string;
 
+  @HostBinding('attr.disabled') get getDisabled() {
+    return this.isDisabled ? true : null;
```

Comment:
Last nit, let's have this equal to `''` on happy case so that html looks like this

```
<button disabled>...</button>
```

instead of 

```
<button disabled="true"></button>
```

Reply:
```suggestion
    return this.isDisabled ? '' : null;
```